probe-rs/src/config/sequences/nxp_armv7m.rs

```diff
@@ -58,6 +58,19 @@ impl MIMXRT10xx {
         }
         Ok(())
     }
+
+    /// Use the boot fuse configuration for FlexRAM.
+    ///
+    /// If the user changed the FlexRAM configuration in software, this will undo
+    /// that configuration, preferring the system's POR FlexRAM state.
+    fn use_boot_fuses_for_flexram(&self, probe: &mut dyn ArmProbe) -> Result<(), ArmError> {
+        const IOMUXC_GPR_GPR16: u64 = 0x400A_C040;
+        let mut gpr16 = probe.read_word_32(IOMUXC_GPR_GPR16)?;
+        gpr16 &= !(1 << 2);
```

Comment:
Can you please introduce a named constant for this bit in both of the `use_boot_fuses_for_flexram` functions?

Reply:
Can do. Thanks for keeping me honest.